src/galaxis-footer/pages/pages.component.tsx

```diff
@@ -1,19 +1,38 @@
-import React, { FC } from "react";
+import React, { FC, useEffect, useState } from "react";
 import { PageModel } from "../../models/page.model";
-import { Box, Typography } from "@mui/material";
+import { Box, Link, Typography } from "@mui/material";
 import CustomTitle from "../title/title.component";
 import StyledWrapper from "./pages.style";
 
 type PagesProps = { pages: PageModel[] };
 
-const Pages: FC<PagesProps> = ({ pages }) => {
+const Pages: FC<PagesProps> = ({ pages: initialPages }) => {
+  const [pages, setPages] = useState<PageModel[]>(initialPages);
+
+  useEffect(() => {
+    const path = window.location.pathname;
+    if (path.includes("engines")) {
+      const updatedPages = initialPages.map((item) => ({
+        ...item,
+        openInNewTab: true,
+      }));
+      setPages(updatedPages);
+    }
+  }, [initialPages]);
+
   return (
     <StyledWrapper>
       <CustomTitle className="title" title="Pages" />
       <Box className="pages--box" data-testid="pages-box">
         {pages.map((page, index) => (
           <Typography className="pages--typography" key={index} variant="main">
-            <a>{page.label}</a>
+            <Link
```

Comment:
I don't see where we handle the link. If we are in the Galaxis website the link should be for example "/privacy-policy" in any other place the link should be "https://galaxis.xyz/privacy-policy".